index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Welcome to Busmall's Market Research Page!</title>
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to BusMall.com!</h1>
+      <h4>Home of your premium shopping experience on wheels.</h4>
+    </header>
+    <main>
+      <section>
+        <h2>Welcome to BusMall.com's Market Research Page</h2>
+        <p>Below you'll find yadda yadda</p>
+      </section>
+      <section>
+        <!-- In this ul will live my 3 pictures -->
+        <ul class="img_container">
+          <li><img class="img_one"></li>
+          <li><img class="img_two"></li>
+          <li><img class="img_three"></li>
+        </ul>
```

Comment:
You're gonna kick yourself, but in your JS you're trying to get element by ID, but none of these list items (nor the list itself) has an ID hahahahaha. I wish github supported emojis...